### ISSUE TITLE
Empty output if used in current dir

### ISSUE THREAD

**Issue by tv42** [mohae/json2go#1]
This is confusing behavior:

``` console
$ ls -lh
total 0
$ mkdir example
$ curl -s https://api.github.com/repos/mohae/json2go | json2go -o example/github.go -w -a -n repo
$ ls -lh example/
total 16K
-rw-r--r-- 1 tv tv 4.3K Mar 10 09:28 github.go
-rw-r--r-- 1 tv tv 4.8K Mar 10 09:28 github.json
$ rm -rf example/
$ curl -s https://api.github.com/repos/mohae/json2go | json2go -o github.go -w -a -n repo
$ ls -lh
total 8.0K
-rw-r--r-- 1 tv tv    0 Mar 10 09:29 github.go
-rw-r--r-- 1 tv tv 4.8K Mar 10 09:29 github.json
$
```

**Comment by pwaller** [comment]
I'm getting empty output whatever I do...

**Comment by pwaller** [comment]
I managed to fix my problem #4. This is unlikely to be the same problem as @tv42 encountered, but could be due to the same underlying cause (missing error return).

**Comment by tv42** [comment]
Tested again and I can no longer reproduce this, with current master.
